subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractContinuousIntegrationTest.groovy

```diff
@@ -147,7 +148,10 @@ ${result.error}
         executer.withStdinPipe()
             .withTasks(tasks)
             .withForceInteractive(true)
+            .withArgument("--info")
             .withArgument("--full-stacktrace")
+            .withArgument("--console")
```

Comment:
Do you mean `--console=plain`? https://docs.gradle.org/current/userguide/command_line_interface.html#sec:command_line_customizing_log_format

Reply:
Unintentional left-overs, will remove them.